packages/node/src/lib/notification-template/notification-template.ts

```diff
@@ -0,0 +1,90 @@
+import { AxiosInstance, AxiosResponse } from 'axios';
```

Comment:
```suggestion
import axios from 'axios';
```
Won't be used in the code but required by `WithHttp` to get types